gefserver/pier/staging.go

```diff
@@ -57,6 +57,8 @@ func (p *Pier) DownStreamContainerFile(volumeID string, fileLocation string, w h
 
 	tarBallReader := tar.NewReader(tarStream)
 	header, err := tarBallReader.Next()
+	//defer p.docker.client.RemoveContainer(string(containerID))
+	defer p.docker.client.ForceRemoveContainerOrSwarmService(string(containerID))
```

Comment:
Since we're deferring this check the error returned is not caught; we should print the error if there is any, like this:
````
defer func() {
    err := p.docker.client.ForceRemoveContainerOrSwarmService(string(containerID))
    if err != nil { log.Println("error while forcefully removing container in DownStreamContainerFile", err) }
}
````